### ISSUE TITLE
Can match "Figure" but not "Figure 1" in search bar

### ISSUE THREAD

**Issue by ChenJiuu** [davidrthorn/cross_reference#74]
When I create links, I cannot search them again in the search bar.

## Search "Figure"
![image](https://user-images.githubusercontent.com/50128686/173528580-7f3aa166-1fa6-4040-9171-5729a5d8f4c1.png)

## Search "Figure " (a space after)
![image](https://user-images.githubusercontent.com/50128686/173528799-b00ea2a5-488d-44da-a615-84eb17709a75.png)

Is there any way to fix this?

If not, is there a way to remove all links and make them back to the normal text again?

**Comment by ChenJiuu** [comment]
Yes, I tried reopening and searching in both chrome and safari, but it doesn't work.

In addition, the sidebar of the configuration can't open in chrome.
![image](https://user-images.githubusercontent.com/50128686/173532220-24459ac4-8dba-44f1-a981-a23784cb262e.png)

but can be opened in safari.
![image](https://user-images.githubusercontent.com/50128686/173532343-053c7871-9bcf-429d-8101-a8442fb5a7fb.png)

**Comment by ChenJiuu** [comment]
I think the problem could be due to the space in the configuration sidebar being different from whitespace in google doc.
![image](https://user-images.githubusercontent.com/50128686/173544953-7940ce63-a3a1-4602-a751-ccccc914fc5b.png)

There won't be a  problem if I don't use whitespace.

Is space encoded differently in Cross Reference?

**Comment by davidrthorn** [comment]
Ah yes it uses non breaking space for obvious reasons. Have you tried using
search within Google docs rather than browser?

**Comment by ChenJiuu** [comment]
Yes, I tried, but it didn't work either.

![image](https://user-images.githubusercontent.com/50128686/173546211-6b467655-7ab9-4238-984d-7292c8cbf540.png)

**Comment by jronley** [comment]
I know it has been two years, but maybe someone else will find this useful! I found a workaround - just search "Figure" followed by \s and your figure number. Enable the "Match using regular expressions" option.
![FigureSearch](https://github.com/user-attachments/assets/bf35f535-3bd0-4c48-8fa6-23eb244f43ac)
